index.html

```diff
@@ -137,13 +137,16 @@ <h1>About Myself</h1>
 			<div id="list">
 				<ul>
 					<li> <h2>Languages</h2></li>
-					<div id="trans">
+					<li >
+						<ul id="trans">
```

Comment:
- [ ] (Optional) It would be nice to make your design for this section look like that of the Figma Template that you chose. Kindly create space between the _**About Yourself**_ div and the languages/framework div. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/160239425-a7e37f07-6b1f-41a8-8f57-9abc0bcddba9.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/160239444-d0c37248-922b-49c6-a51c-6cf237fa442d.png)